packages/lesswrong/client/ga.js

```diff
@@ -20,10 +20,11 @@ function googleTagManagerInit() {
 addInitFunction(googleTagManagerInit)
 
 const identifyFullStoryUser = (currentUser) => {
-  const { karma = 0, afKarma = 0, frontpagePostCount = 0, voteCount = 0, createdAt } = currentUser
-  const fullStoryData = { karma, afKarma, frontpagePostCount, voteCount, createdAt }
+  const { karma = 0, afKarma = 0, frontpagePostCount = 0, voteCount = 0, createdAt, username, displayName: lWDisplayName } = currentUser
```

Comment:
I'm confused about where lwDisplayName comes from

Reply:
This is the line where it is created

Reply:
That's how you rename variables in deconstructions. 

`const { a: b } = {a: "asdasd"}`

creates a new variable b = "asdasd"

Reply:
I quite dislike this syntax and would prefer splitting onto a separate line (ie `const lwDisplayName = currentUser.displayName`).